src/core/StelTextureMgr.cpp

```diff
@@ -38,8 +38,8 @@ StelTextureMgr::StelTextureMgr(QObject *parent)
 	: QObject(parent), glMemoryUsage(0), loaderThreadPool(new QThreadPool(this))
 {
 #ifdef Q_PROCESSOR_X86_64
-	//allow up to 4 textures to be loaded in parallel on 64 bit
-	loaderThreadPool->setMaxThreadCount(std::min(4,QThread::idealThreadCount()));
+	//allow up to 16 textures to be loaded in parallel on 64 bit
```

Comment:
This should be a separate commit, it's unrelated to decimation and may hide if we look for a change that broke something.

Reply:
OK,. Sometimes I find small things that just slip in. I would usually forget about them...